soap/README.md

```
@@ -0,0 +1,35 @@
+# اپلیکیشن SOAP
+
+این اپ مسئول تولید و مدیریت گزارش‌های پزشکی استاندارد به فرمت SOAP است.
+
+## وابستگی‌ها
+- Django REST framework
+- Simple JWT
+
+توجه: طبق دستور، تنظیمات اپ در داخل اپ قرار دارد (`soap/settings.py`) و نیازی به تغییر `helssa/urls.py` نیست.
+
+## Endpoints (درون اپ)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)